source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.2
+----------------
+
+*Released August 22, 2023*
+
+New features: 
+
+- Show insights for unbound arrays (:issue:`COMPASS-6836`).
+- Use modal to highlight legacy connections (:issue:`COMPASS-7072`).
+- Automatically add ``{ }`` to Find queries (:issue:`COMPASS-6530`).
+- Show list of legacy connections. Legacy connections refer to connection 
+  formats that are no longer supported beyond |compass| v1.39.0 
```

Comment:
Super not sure if it's worth clarifying it too much, I think my only small concern is that otherwise users might panic that something is wrong with their servers, so I want to make sure they understand that it's just some compass internals that are "legacy". I have a suggestion for a more specific wording, but leave to you to decide if it's worth rewording like that

```suggestion
- Show list of legacy connections. Legacy connections refer to internal connection 
  formats stored on disk that are no longer supported beyond |compass| v1.39.0 
```

Reply:
Ah thanks for pointing that out! Definitely worth updating the copy to something like this given it gives a clearer note on what legacy connection actually is.